src/mps.jl

```diff
@@ -0,0 +1,119 @@
+# add package using "add https://github.com/AdamSmith-physics/MatrixProductStates.jl#main"
+using MatrixProductStates
+using LinearAlgebra
+
+
+#should be added to MatrixProductStates.jl
+function Base.copy(psi::MPS)
+    return MPS(psi.d, psi.N, copy(psi.tensors), psi.centre, psi.chiMax, psi.threshold)
+end
```

Comment:
You might actually want `deepcopy` here instead of `copy` as `copy` is meant to be a shallow copy. You might not need to define your own method here.